docs/extensions/my-first-extension/_posts/1970-01-04-CreatingShortcutAndScreen.md

```diff
@@ -122,52 +81,71 @@ export const themes = extension.themes;
 
 On the other hand, `app/extension.js` is managed by the CLI and you should not change it. When creating screens, the CLI writes their location in `app/extension.js` which are exported in `app/index.js`.
 
-Upload your extension:
+## Previewing Extension Code Changes
+
+We already did this in Getting Started, but let's elaborate on this. Since the app is managed through the Builder, we needed to `push` the extension to Shoutem and `install` it into our app so we can use and preview it in the Builder:
 
 ```ShellSession
 $ shoutem push
 Uploading `Restaurants` extension to Shoutem...
 Success!
-```
-
-Open the `Add screen` modal in the Builder and select the `Custom` category. You can now see your `Restaurants` starting screen (the shortcut).
+$ shoutem install
```

Comment:
This is not needed. Extensions are already installed into the app in Getting Started. However, it would be also good to say that it was already installed, and we only need to install it once and it will get all the updates on the consequent `shoutem push` commands.